tests/sync-test.js

```diff
@@ -85,6 +87,98 @@ describe("messaging between master and client", () => {
 
 });
 
+    
+
+describe("lively2lively backchannel tests", function() {
+  this.timeout(1*1000)
+  before(async() =>{
+    testingClient = await L2LClient.forceNew({});
+    await testingClient.whenRegistered(300);
+  })
+  after(async() => {
+    await testingClient.remove();
+  })
+  beforeEach(async () => setup(2));
+  afterEach(async () => teardown());
+
+  it("ensure clients have l2l clients", async () => {
+    
+    var {world1, masterWorld, client1, master} = state;    
+    var testChannel = new L2LChannel(client1, "receiveOpsFromMaster", master, "receiveOpsFromClient")
+   
+    expect((testChannel.senderRecvrA.l2lclient) &&  (testChannel.senderRecvrA.l2lclient instanceof L2LClient)).equals(true,'client A not L2LClient')
+    expect((testChannel.senderRecvrB.l2lclient) &&  (testChannel.senderRecvrB.l2lclient instanceof L2LClient)).equals(true,'client B not L2LClient')
+
+    await testChannel.senderRecvrA.l2lclient.whenRegistered(300)
+    await testChannel.senderRecvrB.l2lclient.whenRegistered(300)
+    
+    window.testChannel = testChannel;
```

Comment:
Can you clean up debug code? Leaving globals and such around might lead to strange effects